src/journal/journald.c

```diff
@@ -52,7 +52,7 @@ int main(int argc, char *argv[]) {
                 goto finish;
 
         server_vacuum(&server, false);
-        server_flush_to_var(&server);
+        server_flush_to_var(&server, true);
```

Comment:
Well, the real fix for #1397 should
1) create `/var/log/journal`
2) fix `perms/acls/labels`
3) flush `/run/log/journal` to `/var/log/journal`

The current situation:
1. `systemd-journald` recieves `SIGUSR1`
2. `systemd-journald` creates `/var/log/journal/...`
3. `systemd-journald` flushes `/run/log/journal` to `/var/log/journal`

And `systemd-tmpfiles` fixes `acl/perms/labels` somewhere between 2) and 3)

I think this doesn't fix #1397